1.data_preprocessing/1.make_pseudobulks.ipynb

```diff
@@ -0,0 +1,927 @@
+{
```

Comment:
<p>for these filters (200 genes, &gt;3 cells, and &gt;5% MT reads) can you add plots of the before and after to show that these are reasonable cutoffs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NDavidsonLab/BuDDI_HGSC_schildkraut_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>